lib/logic/models/pull-request.js

```diff
@@ -117,10 +133,6 @@ PullRequest.prototype.getHeadSha = function() {
     return this.headSha;
 };
 
-PullRequest.prototype.setActions = function(actions) {
```

Comment:
actions is now set by default when calling setLineComments, and nothing is calling setActions